docusaurus/docs/learn/core-concepts/services/overview.mdx

```diff
@@ -143,3 +143,393 @@ This strategy drives costs in the same way as the `smartrouting` strategy. Howev
 ##### `random`
 This strategy does not change terminator weights. It does simple random selection across all terminators of the highest precedence. 
 
+## Service Hosting
+
+### `host.v2` Configuration
+
+The `host.v2` configuration type defines how edge routers or tunnelers should create terminators and health checks for
+the application endpoints that provide a service.
+
+### Terminators
+
+Each config can have multiple terminators. Terminators have the following properties:
+
+If your service is for a specific server, then you will provide the following three attributes:
```

Comment:
your/you still appears :) I know it's hard to eliminate them, but they stick out to me when i read 'factual' type doc